indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1126,7 +1131,13 @@ private Map<String, TaskReport> getTaskCompletionReports(@Nullable String errorM
                 getTaskCompletionRowStats(),
                 errorMsg,
                 errorMsg == null,
-                handoffWaitMs
+                handoffWaitMs,
+                partitionsThroughput.entrySet()
```

Comment:
For readability and homogeneity with other fields, put this in a separate method `getProcessedPartitionStats` or something.